distribution/pom.xml

```diff
@@ -81,6 +81,17 @@
                 <module>downloads</module>
             </modules>
         </profile>
+        <profile>
```

Comment:
What's the reason for including this in a profile? That would mean we'd need to include this profile as well when doing a release